### DISCUSSION TITLE
Error: list codespaces request failed with status 401

### DISCUSSION THREAD

**Discussion by anay-208** [community/community#128162] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

I'm Trying to use codespace, However, I'm unable to access it. This is what I see
![image](https://github.com/community/community/assets/96932471/745007e7-169a-4c5f-9bfd-3cc8729b5ad3)

And I also have codespaces in my account.

Upon seeing the output of codespace,  I see this error
```
[Client I] GitHub Codespaces version 1.16.22
[Client I] isRunningInWeb: "false"
[Client V] Initializing authentication
[API I] Successfully signed in
[API E] listCodespaces error. Correlation ID: 0A96:10C392:1D7A124:1DFC948:666890B9
[CodespaceInfo E] Failed to fetch codespaces> object #0: list codespaces request failed with status 401
Error: list codespaces request failed with status 401
...
```

If I try open on Vscode, even that doesn't work. 
However, If I try to open on browser. it works perfectly fine.

What can I do to fix this? Github copilot works perfectly fine!

I've tried deauthorizing vscode directly from authorized apps on github

**Comment by anay-208** [comment, marked as answer]
I had to press the account icon above settings, on bottom left hand side corner, and logout from github account, then relogin.

**Reply by Svenlaa** [reply]
thank you! I had the same issue. could not find a way to sign out from through Ctrl + Shift + P :(
This also happened to me when I tried to access a Codespace on VSCode (program) from the GitHub UI. Re-logging-in through the account menu fixes it!
